kola/tests/ignition/resource.go

```diff
@@ -112,21 +112,15 @@ func init() {
 		ClusterSize: 1,
 		Platforms:   []string{"aws"},
 		UserData: conf.Ignition(`{
-		  "ignition": {
-		      "version": "2.1.0"
-		  },
-		  "storage": {
-		      "files": [
-			  {
-			      "filesystem": "root",
-			      "path": "/resource/s3-auth",
-			      "contents": {
-				  "source": "s3://kola-fixtures/resources/authenticated"
```

Comment:
As discussed OOB, please leave this part of the test in place.

Also, the commit message needs updated.

Reply:
Re-added this portion of the test (along with the validation) and updated the commit message. It looks like my reformatting of the text spacing made github think it was entirely new.